src/ontology/config/mondo-match-rules.yaml

```diff
@@ -0,0 +1,63 @@
+rules:
```

Comment:
looks great! Did you update teh table with that yet?

Reply:
Yes.